cpp/player.h

```diff
@@ -0,0 +1,37 @@
+// Copyright 2015-2015 the openage authors. See copying.md for legal info.
+
+#ifndef OPENAGE_PLAYER_H_
+#define OPENAGE_PLAYER_H_
+
+#include <string>
+#include <unordered_map>
+
+#include "unit/resource.h"
+
+namespace openage {
+
+class Player {
+public:
+	Player(unsigned int number);
+
+	const unsigned int player_number; 
+
+	/**
+	 * would be better to have rgb color value
+	 */
+	const unsigned int color;
```

Comment:
i planned to support rgb-teamcolors, but our recoloring shader has to be much more sophisticated for that.